docs/User-Guide_Getting-Started.md

```diff
@@ -55,17 +60,31 @@ It is safe to ignore the message `WARNING: This key is not certified with a trus
 
 ### How to check download integrity?
 
-Since it might happen that your download got somehow corrupted we integrate a checksum/hash for the image. After uncompressing the download you can compare the image's SHA-256 hash with the one contained in the `sha256sum.sha` file. On Windows you can use [7-Zip's built-in hash functionality](https://superuser.com/a/1024913) to display the SHA256 hash while on Linux/macOS you would do this
+Since it might happen that your download got somehow corrupted we integrate a checksum/hash for the image.  You can compare the image's SHA-256 hash with the one contained in the `sha256sum.sha` file. 
+
+On Windows, you can use the Get-Filehash program in Windows Powershell
+to calculate  the SHA256 hash  and then compare it to the hash in the .sha file you have downloaded.
+ 	
+	get-filehash Armbian_20.05.0_Cubox-i_focal_current_5.6.15.img.xz
+	
+	# response
+	Algorithm       Hash                                                                   Path
+	---------       ----                                                                   ----
+	SHA256          B32B65E4E7E06FE042B74ED05433CE97055952721277E38FC766E7F035A93892       C:\Users\....
+	
 
-	shasum -a 256 -c Armbian_*.img.sha Armbian_*.img 
+while on Linux/macOS, in the directory in which you have downloaded the files ,you would do this
+
+	shasum -a 256 -c Armbian_*.img.sha Armbian_*.img.xz
+	#good response
 	Armbian_5.35_Clearfogpro_Debian_stretch_next_4.13.16.img: OK
-	^C
+
 
 ### How to prepare a SD card?
 
 **Important note:** Make sure you use a **good, reliable and fast** SD card. If you encounter boot or stability troubles in over 95 percent of the time it is either insufficient power supply or related to SD card (bad card, bad card reader, something went wrong when burning the image, card too slow to boot -- 'Class 10' highly recommended!). Armbian can simply not run on unreliable hardware so checking your SD card with either [F3](http://oss.digirati.com.br/f3/) or [H2testw](http://www.heise.de/download/h2testw.html) is mandatory if you run in problems. Since [counterfeit SD cards](http://www.happybison.com/reviews/how-to-check-and-spot-fake-micro-sd-card-8/) are still an issue checking with F3/H2testw directly after purchase is **highly recommended**.
 
-7z and zip archives can be uncompressed with [7-Zip](http://www.7-zip.org/) on Windows, [Keka](http://www.kekaosx.com/en/) on OS X and 7z on Linux. Images shall only be written with [Etcher](https://www.etcher.io) on all platforms since unlike other tools Etcher validates burning results **saving you from corrupted SD card contents**.
+Write the xz compressed image   with [Etcher](https://balena.io/etcher) or  [USBImager](https://gitlab.com/bztsrc/usbimager) on all platforms since unlike other tools, either can validate  burning results **saving you from corrupted SD card contents**.
```

Comment:
I would propose to move USBimager to 1st place, Etcher second (its a nice app, but https://github.com/armbian/documentation/issues/66 )